File: Solo Learn/Médio/Pig Latim/pig_latin.py
x = input()

palavras = x.split()
lista = []
for palavra in palavras:
	x = palavra[1:] #1° letra
	y = palavra[:1] #resto da palavra
	
	x += y + "ay"
	lista.append(x)

#acrescentar espacos
frase = ""
for palavra in lista:
	frase += palavra
	frase += " "
	
print(frase)

File: Solo Learn/Médio/Pig Latim/pig_latin.rb

frase = gets.chomp

palavras = frase.split(" ")
lista = []

palavras.each do |palavra|
  x = palavra[1..-1] # 1° letra
  y = palavra[0]    # resto da palavra
  
  x += y + "ay"
  lista << x
end

puts lista.join(" ")